### DISCUSSION TITLE
i have tried to flash the nspanel it didn't work

### DISCUSSION THREAD

**Discussion by Beau11999988** [Blackymas/NSPanel_HA_Blueprint#1617] [Q&A]
I get this message and I don't know what I'm doing wrong.
I tried to flash the nspanel for the first time.

can anyone help me with that.

![Screenshot 2024-01-12 221949](https://github.com/Blackymas/NSPanel_HA_Blueprint/assets/156475589/0b021373-7258-40e3-9adc-bb8d92fe969b)

I placed these files in my home assistant, I highlighted it in red in the Pictures.

nspanel_blueprint.yaml
nspanel_esphome.yaml
nspanel_esphome_addon_climate_cool.yaml
nspanel_esphome_addon_climate_dual.yaml
nspanel_esphome_addon_climate_heat.yaml
nspanel_eu.tft

![Screenshot 2024-01-12 223241](https://github.com/Blackymas/NSPanel_HA_Blueprint/assets/156475589/3b206086-212d-4090-9846-c6b8ee7392db)
![Screenshot 2024-01-12 223447](https://github.com/Blackymas/NSPanel_HA_Blueprint/assets/156475589/8b6a0d24-3e2e-4218-9339-496505aa001f)

**Comment by DonGuaja** [comment]
Also having a similar issue.

`*** [.pioenvs/screen/src/esphome/components/http_request/http_request.o] Source `src/esphome/components/http_request/http_request.cpp' not found, needed by target `.pioenvs/screen/src/esphome/components/http_request/http_request.o'.`

**Comment by DonGuaja** [comment]
When the esphome flash doesn't work you need to "clean build files" on the ESPHome device.

**Comment by Beau11999988** [comment]
I saw that I posted this message in the discussion but I needed to do it in is issues
so that's what I did
here do you have the link
https://github.com/Blackymas/NSPanel_HA_Blueprint/issues/1622

maybe you have the answers to your problem there
